runtime-components/non-repudiation-postgresql/src/test/scala/com/daml/nonrepudiation/postgresql/TablesSpec.scala

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.nonrepudiation.postgresql
+
+import java.time.Instant
+
+import com.daml.ledger.resources.ResourceContext
+import com.daml.nonrepudiation.testing._
+import com.daml.nonrepudiation.{
+  AlgorithmString,
+  FingerprintBytes,
+  PayloadBytes,
+  SignatureBytes,
+  SignedPayload,
+}
+import com.daml.testing.postgresql.PostgresAroundEach
+import org.scalatest.OptionValues
+import org.scalatest.flatspec.AsyncFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+final class TablesSpec
```

Comment:
Totally fine for now but maybe worth considering for the future: property testing can work very well for these type of roundtrip tests.

Reply:
Tracked by https://github.com/digital-asset/daml/issues/8946